Rakefile

```diff
@@ -11,4 +11,15 @@ end
 
 RuboCop::RakeTask.new
 
+task :test do
+  ENV["MD_LOAD_MODE"] = "inline"
+  $stdout.puts "⚙️ Runs rubocop with '-r rubocop_md' options"
```

Comment:
Adding notifications is good idea, but no in the Rakefile. It should be a part of the test suite, so it doesn't depend on the way we run tests (via Rake, via Ruby, or whatever). Having notification here can result in false positives when the actual test suite is not using this env var.